### ISSUE TITLE
Port of MIST for Ethereum CLassic

### ISSUE THREAD

**Issue by Dexaran** [Dexaran/ICO#15]

**Comment by yograterol** [comment]
Can we change the frontend stack for MIST in ETC? Starting for gulp, webpack is better

**Comment by Dexaran** [comment]
Yes, we can change it, but I don't see any reason to start MIST development before restoring GETH compatibility.

**Comment by yograterol** [comment]
@Dexaran why? may be could start at the same time.

**Comment by Dexaran** [comment]
If we will keep a compatibility of GET and GETH CLassic then we can develop Mist that will support both chains together. No need to create a new version or port of Mist in this case.
